src/Hadolint/Rules.hs

```diff
@@ -349,8 +349,8 @@ aptGetPackages args =
         , isAptGetInstall cmd
         ]
   where
-    noOption arg = arg `notElem` options && not ("--" `isPrefixOf` arg)
-    options = ["apt-get", "install", "-d", "-f", "-m", "-q", "-y", "-qq"]
+    noOption arg = arg `notElem` words && not ("-" `isPrefixOf` arg)
+    words = ["apt-get", "install"]
```

Comment:
Would you mind naming this something else? words is a very commonly known function.

You can in-line the list, if coming up with a name is difficult.